### PR TITLE
Fix `prefer-object-spread` in `/website`

### PR DESCRIPTION
**Pull request by fisker**
- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory)
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/pr-XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: website/playground/buttons.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ export class ClipboardButton extends React.Component {
   render() {
     const { children } = this.props;
     const { showTooltip, tooltipText } = this.state;
-    const rest = Object.assign({}, this.props);
+    const rest = { ...this.props };
     delete rest.children;
     delete rest.copy;
```
Comment (thorn0) [inline website/playground/buttons.js]:
```js
const { children, copy, ...rest } = this.props;
```

File: .eslintrc.yml [hunk 2]
```diff
@@ -25,6 +25,9 @@ rules:
   no-inner-declarations: error
   no-unneeded-ternary: error
   no-useless-return: error
+  no-unused-vars:
+    - error
+    - ignoreRestSiblings: true
```
Comment (fisker) [inline .eslintrc.yml]: See https://github.com/prettier/prettier/pull/7269#discussion_r365513743

File: website/playground/buttons.js (hunk 1 above)
Comment (fisker) [reply website/playground/buttons.js]: Fixed

**Comment by fisker** [comment]
You guys prefer

```js
const { children, copy, ...rest } = this.props;
```

or

```js
const { children, copy: _, ...rest } = this.props;
```

?

In this case `copy` is not using

@sosukesuzuki @thorn0 @evilebottnawi

**Comment by evilebottnawi** [comment]
Looks better:
```js
const { children, copy, ...rest } = this.props;
```

**Comment by fisker** [comment]
better, but second can tell `copy` is not using

**Pull request merged**